ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -80,6 +81,7 @@ Item {
 
         text: {
             if (root.communityData.amIBanned) return qsTr("You were banned from community")
+            if (isCommunityRequestLoading) return qsTr("Loading...")
```

Comment:
When it's loading I don't think we want to show any text here. We just put the button in loading state.